src/components/header.jsx

```diff
@@ -1,28 +1,21 @@
-import React, { useState } from "react";
```

Comment:
How will my changes in `header.js` and `footer.js` be transferred over? For `header.js`, it looks like it may be transferred over, but for `footer.js` it seems like you're deleting the file entirely and creating a new one.

Reply:
I renamed all the files that use JSX by using the syntax `git mv oldfile.js newfile.jsx`. Following the airbnb eslint guidelines in many cases two quotation marks are changed to one. Maybe if there were changes throughout the whole file and the file was moved it says it was deleted? Not sure why it considers footer deleted but not header

Reply:
Let's merge this after my PRs are merged.